scripts/test_env.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+Setup or tear down a bare metal
+environment running a debug build of Iroha.
+"""
+
+import argparse
+import ipaddress
+import json
+import logging
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+
+
+class Network:
+    """
+    A network of bootstrapped peers to run on bare metal.
+    """
+    def __init__(self, setup_dir: pathlib.Path, host_addr: ipaddress.IPv4Address, n_peers: int = 4):
+        logging.info("Setting up test environment...")
+        self.setup_dir = setup_dir
+        self.n_peers = n_peers
+        peers_dir = setup_dir.joinpath("peers")
+        os.mkdir(peers_dir)
+        try:
+            shutil.copy2("./configs/peer/config.json", peers_dir)
+            shutil.copy2("./configs/peer/genesis.json", peers_dir)
+            shutil.copy2("./configs/peer/validator.wasm", peers_dir)
+        except FileNotFoundError:
+            logging.error("Some of the config files are missing. \
+                          Please provide them in the `./configs/peer` directory")
+            sys.exit(1)
+        try:
+            shutil.copy2("./target/debug/iroha", peers_dir)
+        except FileNotFoundError:
+            logging.error("Please build the `iroha` by running `cargo build --bin iroha`")
+            sys.exit(1)
+
+        self.peers = [Peer(setup_dir, host_addr, i) for i in range(n_peers)]
+
+        os.environ["IROHA2_CONFIG_PATH"] = str(setup_dir.joinpath("peers/config.json"))
+        os.environ["IROHA2_GENESIS_PATH"] = str(setup_dir.joinpath("peers/genesis.json"))
+        os.environ["IROHA_GENESIS_ACCOUNT_PUBLIC_KEY"] = self.peers[0].public_key
+        os.environ["IROHA_GENESIS_ACCOUNT_PRIVATE_KEY"] = self.peers[0].private_key
+
+        self.generate_trusted_peers()
+    
+    def generate_trusted_peers(self):
+        self.trusted_peers = []
+        for peer in self.peers:
+            self.trusted_peers.append(f'{{"address": "{peer.host_addr}:{peer.p2p_port}", "public_key": "{peer.public_key}"}}')
+        os.environ["SUMERAGI_TRUSTED_PEERS"] = f"[{','.join(self.trusted_peers)}]"
+
+    def run(self):
+        self.peers[0].run(is_genesis=True)
+        for peer in self.peers[1:]:
+            peer.run()
+
+class Peer:
+    """
+    A single bootstrapped peer. Could be a genesis node or a regular peer.
+    Should not be run directly, but rather as a part of a Network.
```

Comment:
>  Should not be run directly,

Which is more than likely going to be ignored by Pythonistas that "know better". At least indicate it with the `_`.